src/wp-content/themes/twentytwentyone/functions.php

```diff
@@ -425,7 +425,7 @@ function twenty_twenty_one_scripts() {
 		get_template_directory_uri() . '/assets/js/polyfills.js',
 		array(),
 		wp_get_theme()->get( 'Version' ),
-		true
+		array( 'in_footer' => true )
```

Comment:
Updated to reflect new function signature given prior specification of `$in_footer` param.